src/PotentialSimulation/Painting/Painting.jl

```diff
@@ -44,18 +44,30 @@ function paint!(pointtypes, potential, face::AbstractSurfacePrimitive{T}, geomet
     widths_ax1 = diff(get_extended_ticks(grid[1]))
     widths_ax2 = diff(get_extended_ticks(grid[2]))
     widths_ax3 = diff(get_extended_ticks(grid[3]))
+    Δw_max_factor = T(1e-2)
```

Comment:
This is just a scaling factor?
I must admit, I haven’t really looked into Δw_mac ...

Reply:
Basically, yes. 
It's a bit tricky to adjust the tolerance in the `in` method for the local grid. 
One parameter, like it is now, is certainly not the perfect solution. 
But it works in most cases.